src/utils/csvDownloadHelpers.ts

```diff
@@ -295,7 +281,7 @@ export async function downloadV1Payouts(
       rows.push([
         date,
         fromWad(p.modCut),
-        fromWad(p.modCutUSD),
+        p.modCutUSD ? fromWad(p.modCutUSD) : 'n/a',
```

Comment:
although it shouldn't ever be in practice, `amountUSD` is nullable in the subgraph and correctly typed via Apollo, so we handle that type case here